### DISCUSSION TITLE
Yang 1.1 and Netconf 1.1 Support in JnC

### DISCUSSION THREAD

**Discussion by asoka083** [tail-f-systems/JNC#119] [General]
Hi , Wanted to check whether any roadmap is there to support Yang 1.1 and Netconf 1.1 in future versions of JnC. Recently I attended a session where I got to know that Netconf 1.1 support is already on the way but Yang 1.1 is not. Any plan to support Yang 1.1 also ?

**Comment by jomiklos** [comment]
Hello,

sorry for delay, my notifications for discussions did not work 😞 

In my understanding, NETCONF 1.1 should work with the latest changes (branch `development` - to be merged into master very shortly), but YANG 1.1 features are not implemented/verified. There is no immediate/explicit plan to add them, but neither is resolution NOT to do it... Are there any specific features that you aim for from the specification?

**Reply by asoka083** [reply]
Thanks for answer. This I asked in case I need to use any yang model in future which is 1.1.

**Reply by martin-volf** [reply]
YANG-1.1 is fully supported in that sense - YANG modules are processed by a pyang plugin and pyang has been adapted to 1.1 long ago. The example 1-interface actually contains modules with `yang-version 1.1`.
